charts/dave/charts/frontend/values.yaml

```
@@ -32,13 +32,12 @@ podSecurityContext:
 
 securityContext:
-  {}
-  # capabilities:
-  #   drop:
-  #   - ALL
-  # readOnlyRootFilesystem: true
-  # runAsNonRoot: true
-  # runAsUser: 1000
+  capabilities:
+    drop:
+    - ALL
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)